### PR TITLE
RRTMG-LW with MCICA

### PR DESCRIPTION
**Pull request by lkluft**

### CODE REVIEW

**Comment by JoyMonteiro** [comment]
Thank you for the PR, Lukas! I will review it and get back soon.

File: climt/_lib/rrtmg_lw/rrtmg_lw_rad.f90 [hunk 1]
```diff
@@ -744,8 +744,8 @@ subroutine inatm (iplon, nlay, icld, iaer, &
       taucmc(:,:) = 0.0_rb
       ciwpmc(:,:) = 0.0_rb
       clwpmc(:,:) = 0.0_rb
-      reicmc(:) = 0.0_rb
-      relqmc(:) = 0.0_rb
+      reicmc(:) = 20.0_rb
```
Comment (JoyMonteiro) [inline climt/_lib/rrtmg_lw/rrtmg_lw_rad.f90]: Why is this needed? This input validation must happen at the python level based on the flag that is
passed in. If this was not caught in earlier version of RRTMG itself, then you should revert this change
and open an issue so that it gets fixed.

**Comment by JoyMonteiro** [comment]
please update HISTORY and AUTHORS as well, so that things are documented and credit is given where due!

File: climt/_lib/rrtmg_lw/rrtmg_lw_rad.f90 (hunk 1 above)
Comment (SallyDa) [reply climt/_lib/rrtmg_lw/rrtmg_lw_rad.f90]: Hi Joy,
Ah, I just ran my test case again and you're right that this isn't necessary, sorry! I was confused, because reicmc and relqmc are defined as local variables in the subroutine rrtmg_lw with an extra model layer compared to reicmcl and relqmcl, which would then have a value of 0, but it seems that the extra layer is not used.

**Comment by lkluft** [comment]
Sorry for the messy update, but we had to do some patchwork on our side 😉

**Comment by JoyMonteiro** [comment]
No issues! That's what a code review is for anyways :) Will go through and get back (or merge) early next week.

**Comment by JoyMonteiro** [comment]
I compiled the code on my machine and was able to successfully run the example. This looks great, thanks again for the PR. I will release a new version of climt with this functionality sometime soon.
